Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```diff
@@ -228,12 +228,6 @@
         layer:
         - MobLayer
   - type: Appearance
-    rotate: true
-    states:
-      Alive:
-        Base: onestar_boss
-      Dead:
-        Base: onestar_boss_wrecked
```

Comment:
The engine PR fixes some appearance related bugs and in the process removes the client/server specific appearance components.

Now that they're gone the yaml linter errors on this prototype because these fields don't actually exist and weren't ever doing anything.